mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule BambooSmtp.Mixfile do
       {:earmark, ">= 1.0.3", only: :dev},
       {:ex_doc, "~> 0.14.5", only: :dev},
       {:excoveralls, "~> 0.6.0", only: :test},
-      {:gen_smtp, "~> 0.11.0"},
+      {:gen_smtp, github: "Vagabond/gen_smtp", branch: "master"},
```

Comment:
I agree with your concern. Using master is not a very good idea, and merging this pull request is only viable once gen_smtp gets a new release.

I made a new [issue](https://github.com/Vagabond/gen_smtp/issues/125) at gen_smtp, so let's hope they are able to make a new release soon :)